tools/others/email_verifier.py

```diff
@@ -7,13 +7,14 @@ class KnockMail(HackingTool):
     TITLE = "Knockmail"
     DESCRIPTION = "KnockMail Tool Verify If Email Exists"
     INSTALL_COMMANDS = [
-        "git clone https://github.com/4w4k3/KnockMail.git",
-        "cd KnockMail;sudo pip install -r requeriments.txt"
+        "sudo git clone https://github.com/heywoodlh/KnockMail.git",
```

Comment:
Why do we need sudo for a git clone? We don't really need it here i guess

Reply:
this is true. I will remove `sudo`